packages/app/src/components/Settings/EditModal.js

```diff
@@ -61,7 +61,17 @@ const EditModal = ({
                         ? new Date(openedOn)
                         : null
                     }
-                    onChange={(value) => handleOpenedOnDateChanges(value)}
+                    onChange={(date) => {
```

Comment:
logic needed so you can no longer submit more than 4 digits for the year here and you'll see this other places